test/Interop/Cxx/class/inheritance/functions-module-interface.swift

```diff
@@ -1,52 +1,94 @@
-// RUN: %target-swift-ide-test -print-module -module-to-print=Functions -I %S/Inputs -source-filename=x -enable-experimental-cxx-interop | %FileCheck %s
+// RUN: %target-swift-ide-test -print-module -print-implicit-attrs -module-to-print=Functions -I %S/Inputs -source-filename=x -enable-experimental-cxx-interop | %FileCheck %s
 
 // CHECK:      struct NonTrivial {
 // CHECK-NEXT:   init()
+// CHECK-NEXT:   @discardableResult
```

Comment:
Seems like we're getting a lot of `@discardableResult` attrs when the base class' member didn't have that attribute. Is it possible that the base class had the attribute but it was false (so it didn't get printed)? Or do you have any idea what's going on here?

Reply:
I think clang importer attaches `@discardableResult` to all non-void methods that aren't annotated with `__attribute__((warn_unused_result))` (see `lib/ClangImporter/ImportDecl.cpp:9247`). The reason they suddenly showed up in the test, is because I added the `-print-implicit-attrs` to check for attributes that are imported as `implicit` by the clang importer (like `@available`).

In order to make the test less cluttered, we could switch to using `CHECK` so we can ignore the lines with `@discardableResult`. Or I could add a new test for inherited attributes. What do you think?

Reply:
Ah, I see now. Thanks for explaining. Make sense. This LGTM.